### PR TITLE
Updated Privacy statement

### LINKED ISSUE #2397: Privacy policy
Draft here: https://www.notion.so/gfsc/PlaceCal-Privacy-Policy-2a8aeef755a64fbdba77d8bac3b0f985?pvs=4

## Acceptance criteria
- [ ] There is a privacy policy linked from the footer
- [ ] The privacy policy reflects accurately what we do

# Implementation plan

- [ ] Dev team sign off draft
- [ ] Content team sign off draft
- [ ] Add link to site footer
- [ ] Add new page for it

### PR DESCRIPTION
**Pull request by ivan-kocienski-gfsc**
Closes #2397

### CODE REVIEW

File: app/views/pages/privacy.html.erb [hunk 1]
```diff
@@ -4,22 +4,78 @@
...
 
-      <h3>Log Files</h3>
-      <p>PlaceCal.org follows a standard procedure of using log files. These files log visitors when they visit websites. All hosting companies do this and a part of hosting services’ analytics. The information collected by log files include internet protocol (IP) addresses, browser type, Internet Service Provider (ISP), date and time stamp, referring/exit pages, and possibly the number of clicks. These are not linked to any information that is personally identifiable. The purpose of the information is for analyzing trends, administering the site, tracking users’ movement on the website, and gathering demographic information.</p><h3>Cookies and Web Beacons</h3><p>Like any other website, PlaceCal.org uses ‘cookies’. These cookies are used to store information including visitors’ preferences, and the pages on the website that the visitor accessed or visited. The information is used to optimize the users’ experience by customizing our web page content based on visitors’ browser type and/or other information.</p>
+    <p>Provisional content</p>
```
Comment (katjam) [inline app/views/pages/privacy.html.erb:8]: This needs to go, right?

File: app/views/pages/privacy.html.erb [hunk 2]
```diff
@@ -4,22 +4,78 @@
...
+    
+    <p>At PlaceCal.org, one of our main priorities is the privacy of our visitors. This Privacy Policy document contains the types of information that are collected and recorded by placecal.org and placecal-staging.org, and how we use it.</p>
```
Comment (katjam) [inline app/views/pages/privacy.html.erb:8]:
```suggestion
    <p>One of our main priorities is the privacy of our visitors. This Privacy Policy document contains the types of information that are collected and recorded by placecal.org and placecal-staging.org, and how we use it.</p>
```

File: app/views/pages/privacy.html.erb (hunk 1 above)
Comment (katjam) [reply app/views/pages/privacy.html.erb]:
```suggestion
```

Comment (ivan-kocienski-gfsc) [reply app/views/pages/privacy.html.erb:8]: Good catch

File: app/views/pages/privacy.html.erb [hunk 3]
```diff
@@ -4,22 +4,76 @@
...
+    <p>Address: 5 Ribston Street, Manchester, England, M15 5RH</p>
+    <p>Email: <a href="mailto:support@placecal.org">support@placecal.org</a></p>
+    
+    <h3>Personal Information</h3>
+    <h4>The type of personal information we collect</h4>
+    <p>We currently collect and process the following information from organisations who sign-up to use PlaceCal’s services, and any individuals who represent them:</p>
+    <ul>
+      <li>Personal identifiers, contacts and characteristics (for example, name and contact details)</li>
+      <li>Email addresses for organisations which may belong to identifiable individuals</li>
+      <li>Phone numbers for organisations which may belong to identifiable individuals</li>
```
Comment (katjam) [inline app/views/pages/privacy.html.erb:22]:
```suggestion
      <li>Phone numbers for organisations which may belong to identifiable individuals</li>
      <li>Addresses for organisations which may belong to identifiable individuals</li>
```

File: app/views/pages/privacy.html.erb [hunk 4]
```diff
@@ -4,22 +4,76 @@
...
+      <li>Addresses for events which may belong to identifiable individuals</li>
+    </ul>
+    <p>We do not collect personal information from users who are browsing the PlaceCal site without an account.</p>
```
Comment (katjam) [inline app/views/pages/privacy.html.erb:25]:
```suggestion
    <p>We do not collect personal information from users who are browsing a PlaceCal site without an account.</p>
```

File: app/views/pages/privacy.html.erb [hunk 5]
```diff
@@ -4,22 +4,76 @@
...
+    
+    <h4>How we get the personal information and why we have it</h4>
+    <p>Most of the personal information we process is provided to us directly by you for the purpose of creating and hosting events on the PlaceCal platform.</p>
+    <p>We use the information that you have given us in order to share this information with those browsing the site and viewing the calendar.</p>
```
Comment (katjam) [inline app/views/pages/privacy.html.erb:29]:
```suggestion
    <p>We use the information that you have given us in order to share this information with those browsing the site and viewing organisations and their calendars.</p>
```

File: app/views/pages/privacy.html.erb [hunk 6]
```diff
@@ -4,22 +4,76 @@
...
+    <p>Under the UK General Data Protection Regulation (UK GDPR), the lawful bases we rely on for processing this information is your consent. </p>
```
Comment (katjam) [inline app/views/pages/privacy.html.erb:30]:
```suggestion
    <p>Under the UK General Data Protection Regulation (UK GDPR), the lawful basis we rely on for processing this information is your consent. </p>
```

File: app/views/pages/privacy.html.erb [hunk 7]
```diff
@@ -4,22 +4,77 @@
...
+    <h4>How we store your personal information</h4>
+    <p>Your information is securely stored on our server in London, UK.</p>
+    <p>We keep personal identifiers, contacts and characteristics (for example, name and contact details), email addresses for organisations which may belong to identifiable individuals, phone numbers for organisations which may belong to identifiable individuals, and addresses for events which may belong to identifiable individuals for as long as this information is relevant – meaning as long as a partner, user or calendar this information is associated with is active. ‘Active’ in this sense refers to being a live account which can be logged into by the user, a partner with an associated calendar, or a calendar with a working source.</p>
```
Comment (katjam) [inline app/views/pages/privacy.html.erb:36]:
```suggestion
    <p>We keep personal identifiers, contacts and characteristics (as described in the "Type of information we collect" section above) for as long as this information is relevant – meaning as long as a partner, user or calendar this information is associated with is active. ‘Active’ in this sense refers to being a live account which can be logged into by the user, a partner with an associated calendar, or a linked calendar with a working source.</p>
```

File: app/views/pages/privacy.html.erb [hunk 8]
```diff
@@ -4,22 +4,77 @@
...
+    <p><strong>Your right to erasure</strong> - You have the right to ask us to erase your personal information in certain circumstances.</p>
+    <p><strong>Your right to restriction of processing</strong> - You have the right to ask us to restrict the processing of your personal information in certain circumstances.</p>
+    <p><strong>Your right to data portability</strong> - You have the right to ask that we transfer the personal information you gave us to another organisation, or to you, in certain circumstances.</p>
+    <p>You are not required to pay any charge for exercising your rights. If you make a request, we have one month to respond to you.</p>
+    <p>Please contact us at <a href="mailto:support@placecal.org">support@placecal.org</a> if you wish to make a request.</p>
+    
+    <h4>How to complain</h4>
+    <p>If you have any concerns about our use of your personal information, you can make a complaint to us at support@placecal.org</p>
+    <p>You can also complain to the ICO if you are unhappy with how we have used your data.</p>
```
Comment (katjam) [inline app/views/pages/privacy.html.erb:52]: @kimadactyl I did a quick search and seems like Geeks have registered with ICO, but Place Health have not. Should I let Peter know? Or someone else?

File: app/views/pages/privacy.html.erb [hunk 9]
```diff
@@ -4,22 +4,77 @@
...
+    <h3>Web Tracking</h3>
+    
+    <h4>Log Files</h4>
+    <p>PlaceCal.org follows a standard procedure of using log files. These files log visitors when they visit websites. All hosting companies do this and a part of hosting services’ analytics. The information collected by log files include internet protocol (IP) addresses, browser type, Internet Service Provider (ISP), date and time stamp, referring/exit pages, and possibly the number of clicks. These are not linked to any information that is personally identifiable. The purpose of the information is for analysing trends, administering the site, tracking users’ movement on the website, and gathering demographic information.</p>
```
Comment (katjam) [inline app/views/pages/privacy.html.erb:66]: Is this true?

File: app/views/pages/privacy.html.erb [hunk 10]
```diff
@@ -4,22 +4,77 @@
...
 
-      <h3>Online Privacy Policy Only</h3>
-      <p>This privacy policy applies only to our online activities and is valid for visitors to our website with regards to the information that they shared and/or collect in PlaceCal.org. This policy is not applicable to any information collected offline or via channels other than this website.</p>
+    <h4>Cookies, Web Beacons and Third Party Privacy Policies</h4>
+
+    <p>PlaceCal doesn’t store first party cookies.</p>
+
+    <p>PlaceCal uses a third party service called Plausible which allows us to track site visits. You can consult Plausible’s Privacy Policy here: <a href="https://plausible.io/data-policy" target="_blank">https://plausible.io/data-policy</a>. Plausible does not use cookies to track your data, nor does it store any personal data or personally identifiable information (PII).</p>
```
Comment (katjam) [inline app/views/pages/privacy.html.erb:72]: We should maybe move this up under the Log files section - or repurpose that section to incorporate this instead?

File: app/views/pages/privacy.html.erb [hunk 11]
```diff
@@ -4,22 +4,77 @@
...
 
-      <h3>Consent</h3>
```
Comment (katjam) [inline app/views/pages/privacy.html.erb:21]: Do we need to add this back in? Sans the T&C part?

File: app/views/pages/privacy.html.erb (hunk 9 above)
Comment (kimadactyl) [reply app/views/pages/privacy.html.erb:66]: not currently but i think it future proofs us a bit and i have no idea what digital ocean capture either.

we do also capture logs if they end up in rollbar. i think its a nice catchall.

File: app/views/pages/privacy.html.erb (hunk 10 above)
Comment (kimadactyl) [reply app/views/pages/privacy.html.erb:72]: sounds like busywork to me :)

**Comment by aaaaargZombies** [comment]
@honor-gfsc  here if you want to see the discussed content

File: app/views/pages/privacy.html.erb [hunk 12]
```diff
@@ -63,18 +63,16 @@
     <h3>Web Tracking</h3>
     
     <h4>Log Files</h4>
-    <p>PlaceCal.org follows a standard procedure of using log files. These files log visitors when they visit websites. All hosting companies do this and a part of hosting services’ analytics. The information collected by log files include internet protocol (IP) addresses, browser type, Internet Service Provider (ISP), date and time stamp, referring/exit pages, and possibly the number of clicks. These are not linked to any information that is personally identifiable. The purpose of the information is for analysing trends, administering the site, tracking users’ movement on the website, and gathering demographic information.</p>
+    <p>PlaceCal.org's hosting server follows a standard procedure of using log files. These files log visitors when they visit websites. All hosting companies do this and a part of hosting services’ analytics. The information collected by log files include internet protocol (IP) addresses, browser type, Internet Service Provider (ISP), date and time stamp. These are not linked to any information that is personally identifiable.</p>
```
Comment (r-ferrier) [inline app/views/pages/privacy.html.erb:66]: grammatical error:

> All hosting companies do this and a part of hosting services’ analytics.

File: app/views/pages/privacy.html.erb [hunk 13]
```diff
@@ -63,18 +63,16 @@
...
 
-    <h4>Cookies, Web Beacons and Third Party Privacy Policies</h4>
+    <p>PlaceCal uses an error logging service called Rollbar. This service logs errors that may occur while you are browsing the site. These are not linked to any informatin that is personally identifiable. The purpose of these logs is to ensure the security and operation of the website is as expected and alert us when it is not.</p>
```
Comment (r-ferrier) [inline app/views/pages/privacy.html.erb:68]: typo! 

> informatin

File: app/views/pages/privacy.html.erb (hunk 12 above)
Comment (r-ferrier) [inline app/views/pages/privacy.html.erb:66]: > PlaceCal.org

should maybe just be 'PlaceCal'

**Review by r-ferrier — CHANGES_REQUESTED** [review]

File: app/views/pages/privacy.html.erb (hunk 12 above)
Comment (katjam) [reply app/views/pages/privacy.html.erb:66]:
```suggestion
    <p>PlaceCal.org's hosting server follows a standard procedure of using log files. These files log visitors when they visit websites. All hosting companies do this as a part of hosting services’ analytics. The information collected by log files include internet protocol (IP) addresses, browser type, Internet Service Provider (ISP), date and time stamp. These are not linked to any information that is personally identifiable.</p>
```

File: app/views/pages/privacy.html.erb (hunk 13 above)
Comment (katjam) [reply app/views/pages/privacy.html.erb:68]:
```suggestion
    <p>PlaceCal uses an error logging service called Rollbar. This service logs errors that may occur while you are browsing the site. These are not linked to any information that is personally identifiable. The purpose of these logs is to ensure the security and operation of the website is as expected and alert us when it is not.</p>
```

Comment (katjam) [reply app/views/pages/privacy.html.erb:68]: You win!

**Review by r-ferrier — APPROVED** [review]
LGTM

**Pull request merged**